### PR TITLE
Fix transitions

### PR DESCRIPTION
**Pull request by larsiusprime**
Setting scroll factors to 0,0

### CODE REVIEW

**Comment by larsiusprime** [comment]
Also fixed horizontal/vertical wipes to use a lot less memory.

**Comment by larsiusprime** [comment]
(assume it's okay to unilaterally push a small change like this that passed travis?)

**Comment by Gama11** [comment]
Sure.

File: flixel/addons/transition/TransitionFade.hx [hunk 1]
```diff
@@ -154,22 +155,26 @@ class TransitionFade extends Transition
...
 			angle = DirY > 0 ? 90 : 270;
-			s.makeGraphic(FlxG.width, FlxG.height * 2, _data.color);
+			s.makeGraphic(1, FlxG.height * 2, _data.color);
 			pixels = s.pixels;
-			var gvert = FlxGradient.createGradientBitmapData(FlxG.width, FlxG.height, [_data.color, FlxColor.TRANSPARENT], 1, angle);
+			var gvert = FlxGradient.createGradientBitmapData(1, FlxG.height, [_data.color, FlxColor.TRANSPARENT], 1, angle);
 			pixels.copyPixels(gvert, gvert.rect, new Point(0, locY));
 			s.pixels = pixels;
+			s.scale.set(FlxG.width, 1.0);
+			s.updateHitbox();
```
Comment (Gama11) [inline flixel/addons/transition/TransitionFade.hx]: Do these sprites need hitboxes for anything?

**Comment by larsiusprime** [comment]
Thx!

File: flixel/addons/transition/TransitionFade.hx (hunk 1 above)
Comment (larsiusprime) [inline flixel/addons/transition/TransitionFade.hx]: Apparently yes. Without that, they don't render in the correct location.

Comment (madnotdead) [inline flixel/addons/transition/TransitionFade.hx]: I have an issue in current stable version when changing the hitbox of the sprite. When it renders the position depend on the edge of the hitbox.. i think is something related

**Pull request merged**